selfdrive/sensord/sensors_qcom2.cc

```diff
@@ -218,5 +218,11 @@ int sensor_loop() {
 
 int main(int argc, char *argv[]) {
   setpriority(PRIO_PROCESS, 0, -18);
+
+  cpu_set_t mask;
+  CPU_ZERO(&mask);
+  CPU_SET(1, &mask);
+  sched_setaffinity(0, sizeof(mask), &mask);
```

Comment:
there's a helper for this in common/util.cc

Reply:
done